core/src/components/divider/divider.common.scss

```
@@ -0,0 +1,41 @@
+@import "../../themes/native/native.globals";
+
+:host {
+  /**
+   * @prop --margin-top: Top margin of the divider
+   * @prop --margin-bottom: Bottom margin of the divider
+   * @prop --padding-end: Right padding if direction is left-to-right, and left padding if direction is right-to-left of the divider
+   * @prop --padding-start: Left padding if direction is left-to-right, and right padding if direction is right-to-left of the divider
+   */
+  --margin-top: 0px;
+  --margin-bottom: 0px;
+  --padding-start: 0px;
+  --padding-end: 0px;
+
+  display: block;
+
+  width: 100%;
+
+  /* stylelint-disable */
+  @include ltr() {
+    padding-right: var(--padding-end);
```

```suggestion
    padding-right: calc(var(--padding-end) + var(--ion-safe-area-right, 0px));
```

I realized that I copied the original style from item, which is slightly different because it has inner padding that accounts for this. However, we also need to account for the right safe area here.